components/StartProbe.vue

```diff
@@ -13,27 +13,14 @@
 		/>
 	</div>
 	<div class="relative">
+		<CopyButton :content="commands[platform][size]"/>
 		<div ref="codeWrapperElem" class="mt-4 box-content overflow-hidden rounded-xl border p-4 pr-0 transition-[height] duration-500 dark:bg-dark-900">
-			<!-- TODO: P3: collapse/expand thing could be a component, collapse/expand is not smooth if triggered immideately after copy click -->
```

Comment:
Collapse/expand is used in only one place right now, lets wait for a few more usages to extract it to the component.